internal/dao/stubs/context_getters.go

```diff
@@ -0,0 +1,45 @@
+package stubs
+
+import (
+	"context"
+	"errors"
+
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+)
+
+type daoStubCtxKeyType int
+
+const (
+	accountCtxKey daoStubCtxKeyType = iota
+	pubkeyCtxKey  daoStubCtxKeyType = iota
+)
+
+var contextReadError = errors.New("the message")
+
+func WithPubkeyDao(parent context.Context, init_store []*models.Pubkey) context.Context {
```

Comment:
```suggestion
func WithPubkeyDao(parent context.Context, initStore []*models.Pubkey) context.Context {
```

I still think that the DAO interface should be utilized to actually hide the implementation of stubs. What if we choose to use map instead of slice? Or some kind of in memory database. If you insist exposing the slice this way tho, I am okay.